index.html

```diff
@@ -0,0 +1,255 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Josh Lee</title>
+
+  <!-- Leave these links here...
+  But this should be in a comment... -->
+  <link href="https://fonts.googleapis.com/css?family=Cormorant|Poppins:300,400" rel="stylesheet"/>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
+  <link type="text/css" rel="stylesheet" href="styles/style.css"/>
+</head>
+<body>
+  <div class="app">
+    <nav class="app__nav">
+      <div class="container">
+        <div class="top-nav">
+          <h1 class="top-nav__left">Josh Lee</h1>
+
+          <div class="top-nav__left">
+            <ul class="nav-list">
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#welcome">Welcome</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#about">About</a>
+              </li>
+              <li class="nav-list__item">
+                  <a class="nav-list__link" href="#quotes">Quotes</a>
+              </li>
+              <li class="nav-list__item">
+                <a class="nav-list__link" href="#photos">Photos</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </nav>
+    <div class="app__content">
+      <section class="section--hero" id="welcome">
```

Comment:
This is missing the `section` class along with `section--hero` which is why the nav bar looks like it's getting squished.